draft-ietf-ace-oscore-profile.xml

```diff
@@ -173,15 +177,15 @@
    | <---------------------------- Access Token ----- |
    |                               + RS Information   |
    | ---- POST /authz-info ---> |                     |
+   |     (access_token, N1)     |                     |
```

Comment:
I am not sure I should add "Content-Format = application/ace+cbor" here in this figure, for two reasons:
1. I am only trying to give a high level overview of what data is sent, without going into the details of how the messages are formatted
2. The picture would become very cluttered because I would have to add "Content-Format=application/ace+cbor" to all the following exchanges, to be consistent:

```
   | ----- POST /token  ----------------------------> |
   |                            |                     |
   | <---------------------------- Access Token ----- |
   |                               + RS Information   |
   | ---- POST /authz-info ---> |                     |
   |     (access_token, N1)     |                     |
   |                            |                     |
   | <--- 2.01 Created (N2) --- |                     |
```

I can add it if you think it would really help the understanding, but I am not convinced.

Reply:
The reason why I think it might be useful is it then gives context to what the content is.  I can live without it.